### PR TITLE
[Dominion] Add country partner logo component

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes #164528283
## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: dominion/dominion_ui/src/components/CountryPartners/PartnerContent.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+});
+
+function PartnerText({ classes, title, description }) {
+  return (
+    <Grid item xs={12} sm={10} md={4} lg={3} xl={3} className={classes.root}>
+      <Typography variant="h4" className={classes.title}>
+        {title}
+      </Typography>
+      <Typography variant="body1">{description}</Typography>
+    </Grid>
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/CountryPartners/PartnerContent.js]: Don't break `Grid` into separate components. This makes the component hard to reusable. This one should just use `div` i.e.
```jsx
<div>
  <Typography ....
    ...
  </Typography>
</div>
```

File: dominion/dominion_ui/src/components/CountryPartners/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      padding: '2rem'
+    }
+  }
+});
+
+function CountryPartners({ classes }) {
+  return (
+    <Grid
+      xs={12}
+      container
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/CountryPartners/index.js:44]: No need for `xs={12}` here, `<Grid container ...` is enough.

File: dominion/dominion_ui/src/components/CountryPartners/index.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  imageGrid: {
+    padding: '1rem',
+    [theme.breakpoints.up('md')]: {
+      padding: '2rem 1rem'
+    }
+  },
+  logoGrid: {
+    [theme.breakpoints.down('md')]: {
+      padding: '2rem'
+    }
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/CountryPartners/index.js:37]: Mobile first design i.e. reverse the `theme.breakpoints.down` to `up`. e.g.
```javascript
logoGrid: {
  padding: '2rem', // Works for xs, sm and md
  [theme.breakpoints.up('lg')]: {
    padding: 0 // Or to whatever it should be for lg and xl screens
  }
}
```

File: dominion/dominion_ui/src/components/CountryPartners/index.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  return (
+    <Grid
+      xs={12}
+      container
+      direction="row"
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <PartnerContent
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/CountryPartners/index.js]: Move the `Grid` layout from the component to here.

File: dominion/dominion_ui/src/components/CountryPartners/index.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        title="Our Partners"
+        description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+      />
+
+      <Grid
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/CountryPartners/index.js:57]: `<Grid item`

**Review by kilemensi — CHANGES_REQUESTED** [review]
Looks good @nerdlet. Just a few code fixes

**Comment by nerdlet** [comment]
@kilemensi Updated the changes but kinda deleted some files that I need to add back. Will ping you when i've added them back

**Comment by nerdlet** [comment]
@kilemensi Done

**Review by kilemensi — APPROVED** [review]

**Pull request merged**